tools/packaging/release/update-repository-version.sh

```diff
@@ -208,6 +239,34 @@ EOT
 		out=""
 		out=$("${hub_bin}" pull-request -b "${target_branch}" -F "${notes_file}" 2>&1) || echo "$out" | grep "A pull request already exists"
 	fi
+
+	if [ "${repo}" == "kata-containers" ] && [ "${target_branch}" == "main" ] && [[ "${new_version}" =~ "rc" ]]; then
+		reverting_kata_deploy_changes_branch="revert-kata-deploy-changes-after-${new_version}-release"
+		git checkout -b "${reverting_kata_deploy_changes_branch}"
+
+		git revert --no-commit ${kata_deploy_commit} >>/dev/null
+		git add tools/packaging/kata-deploy/kata-deploy/base/kata-deploy-stable.yaml
+		git add tools/packaging/kata-deploy/kata-cleanup/base/kata-cleanup-stable.yaml
+		git add tools/packaging/kata-deploy/kata-deploy/base/kata-deploy.yaml
+		git add tools/packaging/kata-deploy/kata-cleanup/base/kata-cleanup.yaml
+
+		commit_msg="$(generate_revert_kata_deploy_commit $new_version)"
+		info "Creating the commit message reverting the kata-deploy changes"
+		git commit -s -m "${commit_msg}"
+
+		echo "${commit_msg}" >"${notes_file}"
+		echo "" >>"${notes_file}"
+		echo "Only merge this commit after ${new_version} release is successfully tagged!" >>"${notes_file}"
+
+		if [[ ${PUSH} == "true" ]]; then
+			info "Push \"${reverting_kata_deploy_changes_branch}\" to fork"
+			${hub_bin} push fork -f "${reverting_kata_deploy_changes_branch}"
+			info "Create \"${reverting_kata_deploy_changes_branch}\" PR"
+			out=""
+			out=$("${hub_bin}" pull-request -b "${target_branch}" -F "${notes_file}" 2>&1) || echo "$out" | grep "A pull request already exists"
```

Comment:
This line could be fragile if `hub(1)` grows i18n powers, so it would be safest to set `LC_ALL=C LANG=C` before the `grep` here.